src/components/Post/style.scss

```
@@ -0,0 +1,48 @@
+@import '../../styles/constants';
+
+.post{
```

```suggestion
.post {
```

Review the entire file to fix similar issues.